saleor/graphql/product/bulk_mutations/product_bulk_create.py

```diff
@@ -447,6 +448,16 @@ def clean_media(cls, media_inputs, product_index, index_error_map):
                     )
                 )
                 continue
+
+            if alt and len(alt) > 250:
```

Comment:
Could be constant, used here and in message

Reply:
added